### PR TITLE
DRY up `explain` docs and clone logic.

### PR DESCRIPTION
**Pull request by igalklebanov**

### CODE REVIEW

File: src/query-builder/delete-query-builder.ts [hunk 1]
```diff
@@ -956,7 +956,7 @@ export class DeleteQueryBuilder<DB, TB extends keyof DB, O>
 
     return result as SimplifyResult<O>
   }
-
+
```
Comment (koskimas) [inline src/query-builder/delete-query-builder.ts:959]: Some extra whitespace here

File: src/operation-node/query-node.ts [hunk 2]
```diff
@@ -19,6 +22,8 @@ type HasJoins = { joins?: ReadonlyArray<JoinNode> }
 type HasWhere = { where?: WhereNode }
 type HasReturning = { returning?: ReturningNode }
 
+export type HasExplain = { explain?: ExplainNode }
```
Comment (koskimas) [inline src/operation-node/query-node.ts:25]: Why export?

Comment (igalklebanov) [reply src/operation-node/query-node.ts:25]: Haven't updated after the other types were un-exported.

File: src/query-builder/delete-query-builder.ts (hunk 1 above)
Comment (igalklebanov) [reply src/query-builder/delete-query-builder.ts:959]: Nice catch!

**Pull request merged**